src/rust/engine/process_execution/bazel_protos/src/gen/mod.rs

```diff
@@ -0,0 +1,11 @@
+// This file is generated. Do not edit.
```

Comment:
If it's generated, it probably shouldn't be checked in. Does it need to be?

Reply:
It does per the comments in [`.gitignore`](https://github.com/pantsbuild/pants/pull/5742/files#diff-1a11d0be8560f1034703f6a5177695ed). See what you think of that explanation / state of affairs.

Reply:
I'm tempted, if we're going to start checking in generated code, to just check in the generated protobuf structs, and side-step all of these issues. Seems like it would probably prevent a lot of complexity and hassle. WDYT?